raiden/transfer/channel.py

```diff
@@ -190,18 +200,18 @@ def is_channel_usable_for_mediation(
     secret on-chain.
     """
 
-    channel_usable = is_channel_usable_for_new_transfer(channel_state, transfer_amount)
-    lock_timeout_valid = (
-        lock_timeout > 0
-        and channel_state.settle_timeout >= lock_timeout
-        and channel_state.reveal_timeout < lock_timeout
```

Comment:
These checks are important. A lock that does not respect these constraints is not safe to be used. For more details check the function docstring.

Reply:
This is checked as part of `is_channel_usable_for_new_transfer` which is being called by `is_channel_usable_for_mediation` and also in the case of initiator.

In the initiator case the value can be None but in case of mediation it must be a positive number within the allowed range.